src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 40 * days;
```

Comment:
1) Consider renaming this variable to `rentalCost`
2) no needs to use let, use const